src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -43,7 +43,7 @@ class AccessDecisionManager implements AccessDecisionManagerInterface
     public function __construct($voters = [], $strategy = self::STRATEGY_AFFIRMATIVE, $allowIfAllAbstainDecisions = false, $allowIfEqualGrantedDeniedDecisions = true)
     {
         $strategyMethod = 'decide'.ucfirst($strategy);
-        if (!\is_callable([$this, $strategyMethod])) {
+        if (!\is_callable([$this, $strategyMethod]) || 6 === \strlen($strategyMethod)) {
```

Comment:
what's "6"?

Reply:
@nicolas-grekas That refers to the string `decide` from the line before when `$strategy` was an empty string. Personally I would prefer another `if` instead of this tricky to understand condition.

Reply:
Well… `$strategy` is not a string.